spec/controllers/tags_controller_spec.rb

```diff
@@ -183,7 +180,8 @@
         put :update, id: tag, tag: { canonical: false }, commit: "Wrangle"
         tag.reload
         expect(tag.canonical?).to be_falsy
-        it_redirects_to wrangle_tag_path(tag, page: 1, sort_column: "name", sort_direction: "ASC")
+        it_redirects_to_with_notice wrangle_tag_path(tag, page: 1, sort_column: "name", sort_direction: "ASC"),
```

Comment:
Can this be written in a style that's consistent with all the other tests? That is: 
```
it_redirects_to_with_notice(wrangle_tag_path(tag, page: 1, sort_column: "name", sort_direction: "ASC"),
                                    "Tag was updated.")
```